00-intro.md

```diff
@@ -11,15 +11,15 @@ minutes: 5
 
 At a high level, computers do four things:
 
--   run programs
--   store data
--   communicate with each other
--   interact with us
+-   run programs,
+-   store data,
+-   communicate with each other,
+-   and interact with us.
 
 They can do the last of these in many different ways,
-including direct brain-computer links and speech interfaces.
-Since these are still in their infancy,
-most of us use windows, icons, mice, and pointers.
+including direct brain-computer and speech interfaces.
+Since these interfaces are still in their infancy,
+most of us use computer screens, mice, touchpads and keyboards.
```

Comment:
windows and links are not interfaces in the same sense as mice and keyboards. I replaced "windows and icons" with "computer screens".